src/utils/stark/stark-key.ts

```diff
@@ -73,17 +83,8 @@ export function getPublic(keyPair: ec.KeyPair, compressed = false): string {
   return keyPair.getPublic(compressed, 'hex');
 }
 
-/** @deprecated */
-export function getStarkPublicKey(keyPair: ec.KeyPair): string {
```

Comment:
If you remove the deprecated one, can you rename the `getStarkPublicKeyWithXCoordinate` to be `getStarkPublicKey`?